packages/interactivity/src/hooks.tsx

```diff
@@ -232,6 +232,7 @@ const resolve = ( path: string, namespace: string ) => {
 		if ( e === PENDING_GETTER ) {
 			return PENDING_GETTER;
 		}
+		warn( `Failed to resolve "${ path }" in "${ namespace }" namespace.` );
```

Comment:
You need to wrap this with an `if` to `globalThis.SCRIPT_DEBUG`.

Reply:
`warn` already checks for `globalThis.SCRIPT_DEBUG` internally.

https://github.com/WordPress/gutenberg/blob/8474a119f88f1e89d0867d0b1f624111e557692c/packages/interactivity/src/utils.ts#L366-L384